server/locale/DE/Pi30/mutations/pi30.js

```diff
@@ -0,0 +1,148 @@
+import Logo from '../../../../message/logos';
+import {
+    xSmallFallback,
+    textWrap,
+    messageLogoWidth,
+    altNoWrap,
+    setLogoTop,
+    xSmallNoWrap,
+    addPeriod
+} from '../../../../message/mediaQueries';
+import { textLogoMutations } from '../../../../message/logoMutations';
+
+export default {
+    'layout:text': [
+        [
+            'default',
+            ({ textSize }) => ({
+                styles: [
+                    textWrap(textSize * 55, textSize, 'DE'),
+                    xSmallFallback(textSize * 15),
+                    messageLogoWidth(false, textSize * 4, textSize * 1.25),
+                    setLogoTop(textSize * 20),
+                    xSmallNoWrap(textSize * 7.7),
+                    addPeriod(),
+                    `@media screen and 
+                    (min-width: ${textSize * 23.8}px),
+                    (max-width: ${textSize * 21.8}px) {
+                        .message__headline > .tag--medium > span > span.br:last-child {white-space: normal;}
+                    }`
+                ],
+                logo: Logo.PP_PAYPAL.COLOR,
+                headline: [
+                    {
+                        tag: 'medium',
+                        br: ['bei']
+                    },
+                    { tag: 'xsmall', br: ['später'] }
+                ],
+                disclaimer: ['default']
+            })
+        ],
+        [
+            'logo.type:primary && logo.position:right',
+            ({ textSize }) => ({
+                styles: [
+                    textWrap(textSize * 55, textSize, 'DE'),
+                    xSmallFallback(textSize * 15),
+                    setLogoTop(textSize * 42),
+                    messageLogoWidth(textSize * 6, textSize * 4, textSize * 1.25),
+                    addPeriod(),
+                    xSmallNoWrap(textSize * 8.3),
+                    `@media screen and 
+                    (max-width: ${textSize * 18.67}px) {
+                        .message__headline > .tag--medium > span > span.br:nth-child(2) {white-space: nowrap;}
+                    }`
+                ],
+                headline: [{ tag: 'medium', br: ['bei'] }, { tag: 'xsmall' }]
+            })
+        ],
+        [
+            'logo.type:primary && logo.position:top',
+            ({ textSize }) => ({
+                styles: [
+                    textWrap(textSize * 55, textSize, 'DE'),
+                    xSmallFallback(textSize * 14.8),
```

Comment:
```suggestion
                    textWrap(textSize * 55, textSize, 'DE'),
                    xSmallFallback(textSize * 14.8),
```
We won't need these two media queries here.